runtime/optwin.vim

```diff
@@ -1254,6 +1254,8 @@ call <SID>AddOption("isfname", gettext("specifies the characters in a file name"
 call <SID>OptionG("isf", &isf)
 call <SID>AddOption("isident", gettext("specifies the characters in an identifier"))
 call <SID>OptionG("isi", &isi)
+call <SID>AddOption("isexpand", gettext("specifies the characters for expand completion"))
```

Comment:
The meaning of "expand completion" is a bit ambiguous. It can even refer to cmdline completion.